docs/user-guide/story-setting.md

```diff
@@ -1,29 +1,43 @@
+The Story Settings panel allows the editor to customize the theme of the story and configure which additional components should appear to end users in the story view.
 
 The Story settings panel is available in Edit Mode and it can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
 
-
-
 <img src="../img/story-setting/story-setting-1.jpg" class="ms-docimage" style="max-width:400px"/>
 
-The editor can enable or disable and edit the following components:
+## Story Theme
 
-* The **Title** of the story, the default value is the title given to the story's resource in MapStore
+The editor can customize the different components of the story through the following sections:
 
-* The **Theme** to choose the default background and text color of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
+* The **Theme** to choose the default background and text color and the default font of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
 
 <img src="../img/story-setting/theme.gif" class="ms-docimage"/>
 
-* The **Overlay** to choose the default background and text color of the *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
+ * The *Font Family* to choose the default text font present in the whole story: clicking on the search bar a dropdown menu opens to allow selecting the desired font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*): 
+
+<img src="../img/story-setting/font_family.gif" class="ms-docimage"/>
+
+* The **Overlay** to choose the default background and text color of the *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
 
 <img src="../img/story-setting/overlay.gif" class="ms-docimage"/>
 
-* The **Shadow** of the *Content* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content):
+* The *Shadow* of the *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): enabling or disabling the shadow:
```

Comment:
* The **Shadow** of overlay *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): enabling or disabling the shadow